Search/Results.swift

```diff
@@ -1,21 +0,0 @@
-//
```

Comment:
This was auto-generated, but is not required. Think that's a bug in SwiftyJSONAccelerator.